src/style.css

```diff
@@ -1,3 +1,83 @@
-body {
+.body {
+  box-sizing: border-box;
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+.ul {
```

Comment:
Change '.ul' selector to '.nav__list' to match the class name used in the HTML file for the navigation list.
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/c0f910cd-5dae-4432-8e46-1e94441dae55)